src/core/TrainController.hpp

```diff
@@ -33,15 +33,15 @@ namespace godot {
             double max_velocity = 0.0;
             int radio_channel = 0;
             int radio_channel_min = 0;
-            int radio_channel_max = 10;
+            int radio_channel_max = 10; //TODO: Move these to consts to avoid "magic numbers"?
```

Comment:
It's a configuration value.
Radio will be cleaned and moved to a separate node in #33